python/mxnet/onnx/README.md

```diff
@@ -0,0 +1,423 @@
+|resnest200|
+|resnest269|
+|resnet18_v1|
+|resnet18_v1b_0.89|
+|resnet18_v2|
+|resnet34_v1|
+|resnet34_v2|
+|resnet50_v1|
+|resnet50_v1d_0.86|
+|resnet50_v1d_0.48|
+|resnet50_v1d_0.37|
+|resnet50_v1d_0.11|
+|resnet50_v2|
+|resnet101_v1|
+|resnet101_v1d_0.76|
+|resnet101_v1d_0.73|
+|resnet101_v2|
+|resnet152_v1|
+|resnet152_v2|
+|resnext50_32x4d|
+|resnext101_32x4d|
+|resnext101_64x4d|
+|senet_154|
+|se_resnext101_32x4d|
+|se_resnext101_64x4d|
+|se_resnext50_32x4d|
+|squeezenet1.0|
+|squeezenet1.1|
+|vgg11|
+|vgg11_bn|
+|vgg13|
+|vgg13_bn|
+|vgg16|
+|vgg16_bn|
+|vgg19|
+|vgg19_bn|
+|xception|
+|inceptionv3|
+
+|Object Detection|
+|:-|
+|center_net_resnet18_v1b_voc|
+|center_net_resnet50_v1b_voc|
+|center_net_resnet101_v1b_voc|
+|center_net_resnet18_v1b_coco|
+|center_net_resnet50_v1b_coco|
+|center_net_resnet101_v1b_coco|
+|ssd_300_vgg16_atrous_voc|
+|ssd_512_vgg16_atrous_voc|
+|ssd_512_resnet50_v1_voc|
+|ssd_512_mobilenet1.0_voc|
+|faster_rcnn_resnet50_v1b_voc|
+|yolo3_darknet53_voc|
+|yolo3_mobilenet1.0_voc|
+|ssd_300_vgg16_atrous_coco|
+|ssd_512_vgg16_atrous_coco|
+|ssd_300_resnet34_v1b_coco|
+|ssd_512_resnet50_v1_coco|
+|ssd_512_mobilenet1.0_coco|
+|faster_rcnn_resnet50_v1b_coco|
+|faster_rcnn_resnet101_v1d_coco|
+|yolo3_darknet53_coco|
+|yolo3_mobilenet1.0_coco|
+|faster_rcnn_fpn_resnet50_v1b_coco|
+|faster_rcnn_fpn_resnet101_v1d_coco|
+|mask_rcnn_fpn_resnet18_v1b_coco|
+|mask_rcnn_resnet18_v1b_coco|
+|mask_rcnn_resnet50_v1b_coco|
+|mask_rcnn_resnet101_v1d_coco|
+|mask_rcnn_fpn_resnet50_v1b_coco|
+|mask_rcnn_fpn_resnet101_v1d_coco|
+
+|Image Segmentation|
+|:-|
+|fcn_resnet50_ade|
+|fcn_resnet101_ade|
+|deeplab_resnet50_ade|
+|deeplab_resnet101_ade|
+|deeplab_resnest50_ade|
+|deeplab_resnest101_ade|
+|deeplab_resnest269_ade|
+|fcn_resnet101_coco|
+|deeplab_resnet101_coco|
+|fcn_resnet101_voc|
+|deeplab_resnet101_voc|
+|deeplab_resnet152_voc|
+|deeplab_resnet50_citys|
+|deeplab_resnet101_citys|
+|deeplab_v3b_plus_wideresnet_citys|
+|danet_resnet50_citys|
+|danet_resnet101_citys|
+
+|Pose Estimation|
+|:-|
+|simple_pose_resnet18_v1b|
+|simple_pose_resnet50_v1b|
+|simple_pose_resnet50_v1d|
+|simple_pose_resnet101_v1b|
+|simple_pose_resnet101_v1d|
+|simple_pose_resnet152_v1b|
+|simple_pose_resnet152_v1d|
+|alpha_pose_resnet101_v1b_coco|
+|mobile_pose_resnet18_v1b|
+|mobile_pose_resnet50_v1b|
+|mobile_pose_mobilenet1.0|
+|mobile_pose_mobilenetv2_1.0|
+|mobile_pose_mobilenetv3_large|
+|mobile_pose_mobilenetv3_small|
+
+|Action Estimation|
+|:-|
+|inceptionv1_kinetics400|
+|resnet18_v1b_kinetics400|
+|resnet34_v1b_kinetics400|
+|resnet50_v1b_kinetics400|
+|resnet101_v1b_kinetics400|
+|resnet152_v1b_kinetics400|
+|resnet50_v1b_hmdb51|
+|resnet50_v1b_sthsthv2|
+|vgg16_ucf101|
+|inceptionv3_kinetics400|
+|inceptionv3_ucf101|
+
+### [GluonNLP](https://nlp.gluon.ai/model_zoo/catalog.html) Pretrained Model Support Matrix]
+GluonNLP is a popular NLP toolkit built on top of MXNet. Below is the model support matrix for GluonNLP (v0.10.0) models.
```

Comment:
GluonNLP v0.10 does not support MXNet 2. Have you tried with MXNet 2 / should the GluonNLP version be updated here?

Reply:
I think "forward port gluoncv/gluonnlp model tests once they are compatible with MXNet 2.0" from the PR description addresses my question. So mx2onnx currently supports exporting a symbol obtained via guonnlp 0.10 to onnx, it's just that we need mxnet 1 to export that symbol from gluonnlp 0.10

Reply:
Yes that's correct, let me tweak the doc to note that explicitly thanks!